storage/object/how-to/get-started-edge-services.mdx

```diff
@@ -306,6 +306,37 @@ This allows you to specify the precise objects that you want to clear from the c
 
     Your cache is purged and disabled. Edge Services will now serve content by fetching it from your Object Storage bucket directly. If you reenable your cache at a later point, you will begin with an empty cache.
 
+## How to monitor Edge Services with Scaleway Cockpit
+
+You can view your Edge Services metrics via [Scaleway Cockpit](/observability/cockpit/quickstart/). This allows you to monitor your ingress, egress, request rate and cache hit ratio as well as other metrics, in a convenient managed Grafana dashboard.
+
+Access your Edge Services dashboard in the Scaleway console via the shortcut in Edge Services tab of the bucket in question. Note that you will first need to [activate](https://www.scaleway.com/en/docs/observability/cockpit/how-to/activate-cockpit/) Cockpit, and [create a Grafana user and credentials](https://www.scaleway.com/en/docs/observability/cockpit/how-to/retrieve-grafana-credentials/).
```

Comment:
Leaving a comment here to keep in mind that Cockpit **should** be automatically activated by next week, so no need to activate it.